### PR TITLE
feat: introduce grammar, punctuation and principles to uxcs

### PR DESCRIPTION
**Pull request by rossmoody**
## Description

This PR aims to:

1. Restructure the IA of the UXCS section to align with new content
2. Add the Grammar, Principles, and Punctuation sections to the UXCS section

## Checklist

- [ ] :ok_hand: website updates are Garden Designer approved (add the designer as a reviewer)
- [ ] :black_nib: copy updates are approved (add the content strategist as a reviewer)
- [ ] :wheelchair: analyzed via [axe](https://www.deque.com/axe/) and evaluated using VoiceOver
- [ ] :zap: audited via [web.dev](https://web.dev/measure/) for performance and SEO
- [ ] :memo: tested in Chrome, Firefox, Safari, Edge, and IE11

### CODE REVIEW

File: src/pages/content/principles.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: Principles
+description: >
+  The domain of UX content strategy is all the language in a user interface (UI). UX content
+  strategists design with words to create experiences that are clear, consistent, and inclusive.
+---
+
+| Be clear                                                                        | Be consistent                                                            | Be inclusive                                                                      |
+| ------------------------------------------------------------------------------- | ------------------------------------------------------------------------ | --------------------------------------------------------------------------------- |
+| Clear language helps people feel safe about their actions in a user experience. | Consistent language lowers the friction of learning and navigating a UI. | Inclusive language makes an experience usable for the broadest possible audience. |
```
Comment (jzempel) [inline src/pages/content/principles.mdx:10]: Using minimal MD would make this more readable in context. See https://www.markdownguide.org/extended-syntax/#tables for details. Ex:

```md
| Be clear | Be consistent | Be inclusive |
| --- | --- | --- |
| Clear language ... | Consistent language ... | Inclusive language ... |
```

Comment (jzempel) [reply src/pages/content/principles.mdx:10]: @allisonacs your baseline designs didn't express spacing for tables. Is an update needed?

File: src/pages/content/grammar.mdx [hunk 2]
```diff
@@ -1,12 +1,145 @@
...
+save space, but not at the risk of confusing people. Spell out an acronym on first use and include
+it in parentheses. ​
+
+_A service level agreement (SLA) is a contract between you and your customers._
+
+### Bold and italics
+
+Use bold to refer to UI elements. Avoid italics or quotation marks. Using bold is a bold statement.
+A little bit goes a long way, so use bold sparingly. ​ Don't get too bold in notifications. Only use
+bold for headings, and not in the description.
```
Comment (jzempel) [inline src/pages/content/grammar.mdx:17]: Is it worth incorporating the technicality that Garden's bold weight is semibold (600)? Might be the wrong place to call this out.

File: src/pages/content/grammar.mdx [hunk 3]
```diff
@@ -1,12 +1,145 @@
...
+
+## Contractions
+
+Contractions are A-OK. They make sentences shorter and easier to read. Avoid them in flows that feel
+formal like billing, security settings, or account recovery.
+
+## Date and time
+
+Use unicode standards for date and time. Refer to your location-specific standards on
+[CLDR](https://zendesk.atlassian.net/wiki/spaces/globalization/pages/536936649/ICU+Intl+CLDR+at+Zendesk#ICU,Intl&CLDRatZendesk-WhyuseCLDR?).
```
Comment (jzempel) [inline src/pages/content/grammar.mdx:47]: No links to internal-only content on Garden.

File: src/pages/content/grammar.mdx [hunk 4]
```diff
@@ -1,12 +1,145 @@
...
+​ Zendesk uses relative time to show elapsed time. Avoid vague terms like “now,” “soon,” “later,” or
+“in a while.”
+
+| When the event happened                             | Example                   |
+| --------------------------------------------------- | ------------------------- |
+| Less than a minute ago today.                       | < 1 minute ago            |
+| More than a minute and less than an hour ago today. | 43 minutes ago 43 min ago |
```
Comment (jzempel) [inline src/pages/content/grammar.mdx:54]: Example seems like a bug.

File: src/pages/content/grammar.mdx [hunk 5]
```diff
@@ -1,12 +1,145 @@
...
+| More than an hour ago today.                        | Today at {time}           |
+| On the previous calendar date.                      | Yesterday at {time}       |
+| Before yesterday and in the current calendar year.  | {date} at {time}          |
+| Any time before the current year.                   | {date} at {time}          |
+
+## Emoji
+
+Save it for Slack. Emoji is rarely used in the UI. Some emojis have double meanings and don’t
```
Comment (jzempel) [inline src/pages/content/grammar.mdx:62]: _Emoji_ used in the wild seems to tend toward plural. Should the wording here be "Emoji are rarely ..."?

Ref example sentences under https://www.lexico.com/en/definition/emoji

File: src/pages/content/grammar.mdx [hunk 6]
```diff
@@ -1,12 +1,145 @@
...
+localize. 👎 😞
+
+## Numbers and currency
+
+Use unicode standards for numbers and currency. Refer to your location-specific standards on
+[CLDR](https://zendesk.atlassian.net/wiki/spaces/globalization/pages/536936649/ICU+Intl+CLDR+at+Zendesk#ICU,Intl&CLDRatZendesk-WhyuseCLDR?).
```
Comment (jzempel) [inline src/pages/content/grammar.mdx:68]: ~CLDR~

File: src/pages/content/punctuation.mdx [hunk 7]
```diff
@@ -1,12 +1,97 @@
...
+| Headings and subheadings | No                                                                                                                                                                                              |
+| Labels                   | No                                                                                                                                                                                              |
+| System messages          | No, unless it's more than one sentence.                                                                                                                                                         |
+| Title                    | No                                                                                                                                                                                              |
+| Tooltip                  | No, unless it's more than one sentence.                                                                                                                                                         |
```
Comment (jzempel) [inline src/pages/content/punctuation.mdx:72]: Is `prettier` markdown doing this super-table treatment? 🤮

File: src/pages/content/punctuation.mdx [hunk 8]
```diff
@@ -1,12 +1,97 @@
...
+
+## Quotation mark
+
+Avoid using quote marks to indicate places and features in the UI. Use bold instead.
+
+<BestPractice>
+  <Do>
+    <p>Use the End chat button to end a conversation.</p>
```
Comment (jzempel) [inline src/pages/content/punctuation.mdx:80]: Is this meant to be bolded?

File: src/pages/content/punctuation.mdx [hunk 9]
```diff
@@ -1,12 +1,97 @@
...
+</BestPractice>
+
+## Semi-colon
+
+No semi-colons in product; that's the way it has to be.
+
+## Slash
+
+Used when indicating URLs and as a date separator. You can also use slashes to mean “per,” but limit
+use to one slash in a phrase, for example: “\$9/agent per month.”
```
Comment (jzempel) [inline src/pages/content/punctuation.mdx:94]: Is the backslash a copy bug?

File: src/pages/content/principles.mdx (hunk 1 above)
Comment (rossmoody) [reply src/pages/content/principles.mdx:10]: I think my `yarn format` [command might be forcing that formatting](https://share.getcloudapp.com/2Nu5PBOn). Should I try something else?

File: src/pages/content/grammar.mdx (hunk 3 above)
Comment (rossmoody) [reply src/pages/content/grammar.mdx:47]: @vsri1137 Please lemme know how ya wanna handle this

File: src/pages/content/grammar.mdx (hunk 4 above)
Comment (rossmoody) [reply src/pages/content/grammar.mdx:54]: I'm not sure what ya mean

File: src/pages/content/punctuation.mdx (hunk 8 above)
Comment (rossmoody) [reply src/pages/content/punctuation.mdx:80]: Yes, good catch

File: src/pages/content/grammar.mdx (hunk 4 above)
Comment (jzempel) [reply src/pages/content/grammar.mdx:54]: "43 minutes ago 43 min ago"

Comment (rossmoody) [reply src/pages/content/grammar.mdx:54]: Good catch, [looks like there's a break in the google doc.](https://share.getcloudapp.com/2Nu5PXXd) Will fix

File: src/pages/content/punctuation.mdx (hunk 9 above)
Comment (rossmoody) [reply src/pages/content/punctuation.mdx:96]: Yeah, I'm a little unsure how that ended up there.

Comment (rossmoody) [reply src/pages/content/punctuation.mdx:96]: Prettier is adding it

Comment (jzempel) [reply src/pages/content/punctuation.mdx:96]: Appears to be some errant regex treatment 🙁 You might need to resort to using `prettier-ignore` 🤦

File: src/pages/content/grammar.mdx (hunk 2 above)
Comment (vsri1137) [reply src/pages/content/grammar.mdx:17]: I don't know if this is the right spot for it.

File: src/pages/content/grammar.mdx (hunk 3 above)
Comment (vsri1137) [reply src/pages/content/grammar.mdx:47]: Remove link. Our team will know what CLDR is.

File: src/pages/content/grammar.mdx (hunk 5 above)
Comment (vsri1137) [reply src/pages/content/grammar.mdx:62]: I'll be darned. So let's make this:

`Emoji are rarely used in the UI.`

File: src/pages/content/grammar.mdx (hunk 6 above)
Comment (vsri1137) [reply src/pages/content/grammar.mdx:68]: Remove link.

File: src/pages/content/principles.mdx (hunk 1 above)
Comment (jzempel) [reply src/pages/content/principles.mdx:10]: Nah, forget about it. The binary nature of `prettier` is making this impossible to control 🙁

File: src/pages/content/grammar.mdx (hunk 5 above)
Comment (jzempel) [reply src/pages/content/grammar.mdx:62]: /cc @vsri1137

File: src/pages/content/grammar.mdx [hunk 10]
```diff
@@ -1,12 +1,146 @@
...
-Coming soon
+## Acronyms and abbreviations
+
+Acronyms are like inside jokes. If you aren’t in the know they don’t make sense. Use acronyms to
+save space, but not at the risk of confusing people. Spell out an acronym on first use and include
+it in parentheses. ​
+
+_A service level agreement (SLA) is a contract between you and your customers._
+
+### Bold and italics
```
Comment (vsri1137) [inline src/pages/content/grammar.mdx:13]: This should be same hierarchy as rest (H2)

File: src/pages/content/punctuation.mdx [hunk 11]
```diff
@@ -1,12 +1,99 @@
...
-Coming soon
+## Ampersand
+
+Avoid ampersands in place of “and.” Don’t use “+” unless you’re opening a coffee shop in Portland.
+
+## Apostrophe
+
+Use apostrophes to denote possession. Avoid apostrophes to denote plurals even for initialisms.
+
+## Comma
```
Comment (vsri1137) [inline src/pages/content/punctuation.mdx:13]: Alphabetical error on my part. Comma should follow Colon.

File: src/pages/content/grammar.mdx [hunk 12]
```diff
@@ -1,12 +1,146 @@
...
+
+Use bold to refer to UI elements. Avoid italics or quotation marks. Using bold is a bold statement.
+A little bit goes a long way, so use bold sparingly. ​ Don't get too bold in notifications. Only use
```
Comment (allisonacs) [inline src/pages/content/grammar.mdx:16]: There is a double space after "sparingly." There are a few other double space instances peppered in here; might try to do a search and replace for `.  ` which I've found to be generally idiot proof.

File: src/pages/content/grammar.mdx [hunk 13]
```diff
@@ -1,12 +1,146 @@
...
+<BestPractice>
+  <Do>
+    <p>
+      Click on <b>Settings</b> in Admin Center
+    </p>
+  </Do>
+  <Dont>
+    <p>Click on “Settings” in Admin Center</p>
+  </Dont>
+</BestPractice>
```
Comment (allisonacs) [inline src/pages/content/grammar.mdx:28]: @rossmoody Noticed that in these examples, the example of what to do/not do is actually in the body of the component, contrasted with our component examples where the example image is above the text content. 

Are you cool with the deviation?

File: src/pages/content/punctuation.mdx [hunk 14]
```diff
@@ -1,12 +1,99 @@
...
-description: >
-  Lorem ipsum dolor sit amet, consectetur adipiscing
-  elit, sed do eiusmod tempor incididunt ut labore et
-  dolore magna aliqua.
 ---
 
-Coming soon
+## Ampersand
+
+Avoid ampersands in place of “and.” Don’t use “+” unless you’re opening a coffee shop in Portland.
```
Comment (allisonacs) [inline src/pages/content/punctuation.mdx:7]: 😂 "unless you're opening a coffee shop in Portland"

File: src/pages/content/punctuation.mdx [hunk 15]
```diff
@@ -1,12 +1,99 @@
...
+## Exclamation point
+
+These are too loud for most of our brand tones. We tend not to use them at all.
+
+## Hyphens and dashes
+
+Err on the side of fewer hyphens and dashes. Use en dashes with no spaces to communicate a range.
+Avoid using en or em dashes to communicate a break in your thoughts.
+
+_This plan affects any customers with 100–150 active agents._
```
Comment (allisonacs) [inline src/pages/content/punctuation.mdx:50]: Feels like this should be a Do example?

File: src/pages/content/punctuation.mdx [hunk 16]
```diff
@@ -1,12 +1,99 @@
...
+
+## Period
+
+Use periods to end full sentences in body copy. Consult this handy chart to find out where we use
+periods.
+
+| Component                | Does it need a period?                                                                                                                                                                          |
+| ------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Breadcrumbs              | No. Bullets should be sentence fragments. If a bullet has more than one sentence, use closing punctuation on each. If a bullet has more than two sentences, it should be a description instead. |
+| Bullets (in a list)      | No                                                                                                                                                                                              |
```
Comment (allisonacs) [inline src/pages/content/punctuation.mdx:65]: Looks like the Bullets answer is currently showing no, and the Breadcrumbs answer is currently showing the Bullet answer.

File: src/pages/content/grammar.mdx (hunk 13 above)
Comment (rossmoody) [reply src/pages/content/grammar.mdx:28]: I'm ok with it. Good opportunity to rethink the component in the future, though.

**Pull request merged**